pkg/monitor/agent.go

```diff
@@ -190,6 +192,18 @@ func (monitor *AgentMonitor) updateOfPortIPAddress(localEndpointInfo map[string]
 	monitor.syncQueue.Add(monitor.Name())
 }
 
+func (monitor *AgentMonitor) periodicallySyncAgentInfo(cycle int, stopChan <-chan struct{}) {
+	ticker := time.NewTicker(time.Duration(cycle) * time.Second)
+	for {
+		select {
+		case <-ticker.C:
+			monitor.syncQueue.Add(monitor.Name())
```

Comment:
Since all agentinfo is generated form monitor.ipCache, maybe we could gather sync request together in same time? That means the syncQueue has only one capacity, like a token bucket.

Reply:
agentinfo 更新由两类事件触发：1）agent 所在 node 上的 ovsdb 信息更新；2）interface 上的 ip 变动； 同步任务是新增的第三类，定时触发更新一次 agentinfo。互相独立。